### ISSUE TITLE
Why gulp?

### ISSUE THREAD

**Issue by ohall** [GRT/onesie-api#1]
@mikeatkins Seems like npm scripts are simpler for doing simple tasks like linting.  
What's the advantage of using gulp here?

**Comment by mikeatkins** [comment]
so far, this: '**/*.js' .. do npm run scripts allow file globbing? if so, I'm ok just using npm.. although I'm not sure we won't end up needing gulp eventually

**Comment by ohall** [comment]
Yeah, globbing should work.  I'm good with either one.  Just curious whether there was some new thinking.  Electrode has moved back to gulp from NPM scripts, but that was bc they needed to do more complex operations.

**Comment by ohall** [comment]
I'll mess with it and file a PR
